dev-env-app/server/src/server.ts

```diff
@@ -55,7 +60,7 @@ function initApp(): BehodlerDevEnvFastifyInstance {
 }
 
 ;(async () => {
-  const fastify = initApp()
+  const fastify =await initApp()
   await fastify.ready()
```

Comment:
Fair enough. I find relying on an eventual await as opposed to explicitly awaiting each time is just a way of playing it safe.  It's akin to functional programming's avoidance of the anti pattern of relying on ambient context. I noticed that debugging blockchain tests that group promises can lead to a lot of non-deterministic chaos because of timing and ordering.